### PR TITLE
net-dns/unbound: add 1.7.2

### PR DESCRIPTION
**Pull request by bes-internal**

### CODE REVIEW

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: obsolete EAPI

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+PYTHON_COMPAT=( python2_7 )
+
+inherit eutils flag-o-matic multilib-minimal python-single-r1 systemd user
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: no need `eutils`

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+MY_P=${PN}-${PV/_/}
+DESCRIPTION="A validating, recursive and caching DNS resolver"
+HOMEPAGE="http://unbound.net/"
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: https and for `SRC_URI` too

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+	# as well as new installs where unbound homedir has just been created
+	if [[ -d "${ROOT}/etc/unbound" ]]; then
+		chown --no-dereference --from=unbound root "${ROOT}/etc/unbound"
+	fi
+
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	# To avoid below error messages, set 'trust-anchor-file' to same value in
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: `default`

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+	# 'auto-trust-anchor-file'.
+	# [23109:0] error: Could not open autotrust file for writing,
+	# /etc/dnssec/root-anchors.txt: Permission denied
+	epatch "${FILESDIR}"/${PN}-1.5.7-trust-anchor-file.patch
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: `eapply`

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+		# http://unbound.nlnetlabs.nl/pipermail/unbound-users/2011-April/001801.html
+		# $(use_enable debug lock-checks) \
+		# $(use_enable debug alloc-checks) \
+		# $(use_enable debug alloc-lite) \
+		# $(use_enable debug alloc-nonregional) \
+}
+
+multilib_src_install_all() {
+	prune_libtool_files --modules
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: replace with find

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+	systemd_newunit "${FILESDIR}"/unbound_at.service "unbound@.service"
+	systemd_dounit "${FILESDIR}"/unbound-anchor.service
+
+	dodoc doc/{README,CREDITS,TODO,Changelog,FEATURES}
+
+	# bug #315519
+	dodoc contrib/unbound_munin_
+
+	docinto selinux
+	dodoc contrib/selinux/*
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: no globs, `dodoc -r`

File: net-dns/unbound/unbound-1.7.2.ebuild [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+
+	exeinto /usr/share/${PN}
+	doexe contrib/update-anchor.sh
+
+	# create space for auto-trust-anchor-file...
+	keepdir /etc/unbound/var
+	# ... and point example config to it
+	sed -i '/# auto-trust-anchor-file:/s,/etc/dnssec/root-anchors.txt,/etc/unbound/var/root-anchors.txt,' "${ED}/etc/unbound/unbound.conf"
```
Comment (Zlogene) [inline net-dns/unbound/unbound-1.7.2.ebuild]: `|| die`

**Comment by mschiff** [comment]
Thanks for your PR. I am the maintainer of unbound in portage and I'd kindly ask you to open bugs at bugs.gentoo.org for version bumps instead.

I have no control of handling your PRs here at gitgub as I do not have any permissions to accept it etc.

Regarding the comments of @Zlogene  I support many of them but not all. (e.g. I'd still stick to earlier EAPI as long this there is no real reason to bump it. Users will be thankfull as this will bring less problems to them when updating older systems for example.

TIA

**Comment by Zlogene** [comment]
@mschiff there _is_ the reason to bump EAPI -- repoman will not accept a new ebuild with deprecated EAPI (which is 5 by now). Morover, current stable is 7, so at least 6 should be used if you have doubts

File: net-dns/unbound/unbound-1.7.2.ebuild (hunk 1 above)
Comment (mschiff) [reply net-dns/unbound/unbound-1.7.2.ebuild]: Ok, should be bumped to 6

**Pull request closed without merging**